### ISSUE TITLE
Parser does not recognize single character table name or alias

### ISSUE THREAD

**Issue by joseffi** [phpmyadmin/sql-parser#327]
When using a single character as a table alias or name, the table.column names are not properly recognized

For example, this SQL:

```sql
SELECT x.asset_id FROM
    (SELECT evt.asset_id FROM evt) x
```
is incorrectly parsed into:

```            [expr] => Array
                (
                    [0] => PhpMyAdmin\SqlParser\Components\Expression Object
                        (
                            [database] => 
                            [table] => 
                            [column] => 
                            [expr] => x.asset_id
                            [alias] => 
                            [function] => 
                            [subquery] => 
                        )

                )
```

But this works:
```sql
SELECT xx.asset_id FROM
    (SELECT evt.asset_id FROM evt) xx
```

parser result: 
```            [expr] => Array
                (
                    [0] => PhpMyAdmin\SqlParser\Components\Expression Object
                        (
                            [database] => 
                            [table] => xx
                            [column] => asset_id
                            [expr] => xx.asset_id
                            [alias] => 
                            [function] => 
                            [subquery] => 
                        )

                )
```

**Comment by williamdes** [comment]
Thank you for reporting this issue !

**Comment by williamdes** [comment]
Fixed by #385
